include/lsst/meas/base/PixelFlags.h

```diff
@@ -74,8 +80,18 @@ class PixelFlagsAlgorithm : public SimpleAlgorithm {
         CR,
         CR_CENTER,
         BAD,
+        CLIPPED,
         N_FLAGS
     };
+    
+    afw::table::Key<afw::table::Flag> _generalFailure;
+    afw::table::Key<afw::table::Flag> _offImageKey;
+    typedef std::map<std::string, afw::table::Key<afw::table::Flag> > KeyMap;
+    KeyMap _predefinedAnyKeys;
+    KeyMap _predefinedCenterKeys;
+    KeyMap _centerKeys;
+    KeyMap _anyKeys;
+
```

Comment:
Do these all need to be public?

Also, is there any benefit to keeping separate maps for the predefined keys and those set in the config?